integration-test.sh

```diff
@@ -15,7 +15,7 @@ export MINIMIZE_DOWNTIME=0
 
 if [[ "$test_option" == "--initial-install" ]]; then
   echo "Testing initial install"
-  source _integration-test/run.sh
+  pytest --reruns 5 _integration-test/run.py
```

Comment:
I would recommend *not* having reruns -- they're a crutch that allows flakiness to creep in and ideally we wouldn't have them in `sentry`

Reply:
Primary reason behind this is because docker compose itself is quite flaky in CI, and I wanted to have a way to rerun  the fixture that brings self-hosted up. 

https://github.com/getsentry/team-ospo/issues/242